diskplorer.py

```diff
@@ -205,7 +209,27 @@ def global_section():
             job_names = generate_job_names(f'job(r_idx={read_iops_step},w_idx={write_bw_step},write_bw={write_bw},r_iops={read_iops})')
             read_iops = max(read_iops, 10)   # no point in a write-only test
             nr_cpus = args.cpus
+            read_offset_str = ''
+
             if write_bw > 0:
+                if args.run_sequential_trim:
+                    out(textwrap.dedent(f'''\
+                        [prepare_data_for_trim(r_idx={read_iops_step},w_idx={write_bw_step},write_bw={write_bw},r_iops={read_iops})]
+                        '''))
+                    out(group_introducer)
+                    out(textwrap.dedent(f'''\
+                        readwrite=write
+                        runtime={args.trim_offset_time_seconds}s
+                        blocksize={args.write_buffer_size}
+                        iodepth={args.write_concurrency}
+                        rate={write_bw}
+                        '''))
+                    read_offset_str = f'offset={write_bw*(args.test_step_time_seconds+args.trim_offset_time_seconds)}'
+                    write_offset_str = f'offset={write_bw*args.trim_offset_time_seconds}'
```

Comment:
It seems, that the offset needs to be aligned. It will be fixed.

When diskplorer.py is run in the current state, sometimes I can see:

```
fio: io_u error on file /dev/nvme1n1: Invalid argument: write offset=163202660, buflen=131072:00s]
fio: io_u error on file /dev/nvme1n1: Invalid argument: write offset=163333732, buflen=131072
fio: io_u error on file /dev/nvme1n1: Invalid argument: read offset=1920005292432, buflen=512
fio: io_u error on file /dev/nvme1n1: Invalid argument: read offset=1209887404944, buflen=512
fio: io_u error on file /dev/nvme1n1: Invalid argument: read offset=15709349264, buflen=512
fio: io_u error on file /dev/nvme1n1: Invalid argument: read offset=224576115088, buflen=512
```

Reply:
Fixed